internal/storage/bucket_handle.go

```diff
@@ -221,8 +221,11 @@ func (b *bucketHandle) ListObjects(ctx context.Context, req *gcs.ListObjectsRequ
 	// Iterating through all the objects in the bucket and one by one adding them to the list.
 	for {
 		var attrs *storage.ObjectAttrs
-		// itr.next returns all the objects present in the bucket. Hence adding a check to break after required number of objects are returned.
-		if len(list.Objects) == req.MaxResults {
```

Comment:
if req.MaxResults=0, why not set it as req.MaxResults=Integer.Max or something. It will avoid this extra condition in loop

Reply:
mybad. i thought 225 line is not removed. LG